docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

```diff
@@ -34,6 +34,8 @@ For more information about cloud-controller-manager, please see:
 
 **NOTE: Now, the openstack-cloud-controller-manager implementation is based on OpenStack Octavia, Neutron-LBaaS has been deprecated in OpenStack since Queens release and no longer maintained in openstack-cloud-controller-manager. So make sure to use Octavia if upgrade to the latest openstack-cloud-controller-manager docker image.**
 
+**NOTE: for test/PoC with only 1 master node environment, you need remove the label `node.kubernetes.io/exclude-from-external-load-balancers` of the master node otherwise the loadbalancer will not be created. search the label [here](https://pkg.go.dev/k8s.io/api/core/v1) for further information.**
```

Comment:
How about we add this to `Limitation` section or in the doc `Exposing applications using services of LoadBalancer type`?